### DISCUSSION TITLE
potential minor documentation issue on custom dialogs

### DISCUSSION THREAD

**Discussion by NomanTrips** [quasarframework/quasar#13262] [General - Components / Directives / etc] [answered]
In the documentation there's a section on how to make a custom dialog pop-up:
https://quasar.dev/quasar-plugins/dialog#invoking-custom-component

In there it reads:
  $q.dialog({
    component: CustomComponent,

This didn't work for me I had to capitalize 'component':
  $q.dialog({
    Component: CustomComponent,

I have no idea why this caused an issue for me but there were no full examples online when searching on google. The error was obscure stating: [Quasar] Incorrectly defined Dialog component

**Comment by MilosPaunovic** [comment, marked as answer]
The code responsible for dialog creation is in this file:
https://github.com/quasarframework/quasar/blob/dev/ui/src/utils/private/global-dialog.js

As you can see, capitalized `Componennt` is not expected anywhere, so it's probably up to something else in your codebase, not the capitalization issue.

**Reply by NomanTrips** [reply]
Thanks, I found out what I was doing wrong: I didn't fully read the documentation and wasn't following the section right below detailing making the custom component. Such as:
https://quasar.dev/quasar-plugins/dialog#sfc-with-script-setup-and-composition-api-variant
I made the mistake because these had the word 'variant' in the title, and I assumed some tooling variant. I thought the component creation wasn't detailed, so I took a guess at it and that was breaking things.
In any case, user error :) .
